src/main/java/nl/tudelft/contextproject/gui/MenuController.java

```
@@ -223,15 +267,212 @@ private void initOtherButtons() {
+            settingsGrid.disableProperty().set(true);
+        } catch (IOException e) {
+            AlertDialog.errorSavingSettings(e);
+        }
+    }
+    
+    /**
+     * Initialises the VLC settings section of the settings menu.
+     * 
+     * @param settings The instance of ApplicationSettings
+     */
+    private void settingsInitVlcSettings(ApplicationSettings settings) {
+        settingsVlcBox.setItems(FXCollections.observableArrayList(
+                "1080p", "720p", "480p"));
+        settingsVlcBox.setTooltip(new Tooltip(
+                "Select the preferred quality for rendering VLC live views"));
+        settingsVlcBox.getSelectionModel().select(settings.getRenderResY() + "p");
+        settingsVlcBox.setOnAction(event -> {
+            String selected = settingsVlcBox.getValue();
+            if (selected != null) {
+                int resY = Integer.parseInt(selected.substring(0, selected.length() - 1));
+                int resX;
+                if (resY == 1080) {
+                    resX = 1920;
+                } else if (resY == 720) {
+                    resX = 1280;
+                } else {
+                    resX = 720;
+                }
+                settings.setRenderResolution(resX, resY);
+            }
+        });
+        
+        settingsVlcLoc.setTooltip(new Tooltip("Current VLC installation location"));
+        if (!ContextTFP.hasVLC()) {
+            settingsVlcLoc.setText("No VLC found.");
+        } else if (settings.getVlcLocation().equals("")) {
+            settingsVlcLoc.setText("Default location");
+            btnChangeVlcLoc.setDisable(true);
+        } else {
+            settingsVlcLoc.setText(settings.getVlcLocation());
+        }
+        
+        btnChangeVlcLoc.setOnAction(event -> {
+            try {
+                AlertDialog.findVlc(((Node) event.getTarget()).getScene().getWindow());
+            } catch (RuntimeException e) {
+                //No VLC has been set.
```

PMD complains about empty catch block, but I think PMD should shut up since you nicely documented when the code would reach this point.